app/build.gradle

```diff
@@ -111,15 +113,7 @@ dependencies {
   androidTestImplementation rootProject.ext.testRules
   androidTestImplementation rootProject.ext.coroutinesTest
   androidTestImplementation rootProject.ext.composeTestJUnit
+  androidTestImplementation rootProject.ext.facebookSnapshotTestingCore
```

Comment:
Why do we still need to have it?

Reply:
we still need direct access for a few things, like setting the custom screenshot names for parameterized tests and setting up the view

Reply:
Let's create a ticket for removing/replacing it

Reply:
we can't :) if we have parameterized tests the files will get overridden without us changing the screenshot name.